dm/proto/dmmaster.proto

```diff
@@ -62,6 +62,22 @@ service Master {
 
     // OfflineWorker offline the dm-workers.
     rpc OfflineWorker(OfflineWorkerRequest) returns(OfflineWorkerResponse) {}
+
+    // ListMember list member master information
+    rpc ListMemberMaster(ListMemberRequest) returns(ListMemberMasterResponse) {}
+
+    // ListMember list member worker information
+    rpc ListMemberWorker(ListMemberRequest) returns(ListMemberWorkerResponse) {}
+
+    // ListMember list member leader information
+    rpc ListMemberLeader(ListMemberRequest) returns(ListMemberLeaderResponse) {}
```

Comment:
Can we combine them into one method?

Reply:
But their return type are different.

Reply:
Is `oneof` can work here? (as we did in `message SubTaskStatus`).

Reply:
Got it. I will have a try.